JS/Lesson34/gulcin/recipe-app/src/App.css

```diff
@@ -10,15 +23,22 @@
   background-color: white;
 }
 
+ul {
+  list-style: none;
+}
+
 .recipes__list {
-  list-style-type: none;
   padding-left: 0;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr 1fr;
+ 
 }
 
-.recipe__list > li {
-border-bottom: 1px solid var(--color-light);
-display: flex;
-justify-content: space-between;
-margin: 10px auto;
+.recipes__list>li {
+  border-bottom: 1px solid gray;
```

Comment:
It's always better to use CSS variables as you use colors in different places. Using color names / codes make maintenance hard. It's even better to use a theme.
In the upcoming lesson we'll see how to use component libraries in react and it will also include the theme, so no need to change colors now but definitely change to the theme colors after that.